config/default.json

```diff
@@ -1,80 +1,93 @@
+        "frameAncestors": ["'none'"],
+        "imgSrc": [
+          "'self'",
+          "data:",
+          "https://www.google-analytics.com",
+          "https://www.googletagmanager.com",
+          "https://raw.githubusercontent.com/hmcts/"
+        ]
+      }
     },
-    "logging": "debug",
-    "idamClient": "xuimowebapp",
-    "microservice": "xui_webapp",
-    "now": "NOW",
-    "oauthCallbackUrl": "oauth2/callback",
-    "protocol": "https",
-    "services": {
-        "idamApi": "https://idam-api.platform.hmcts.net",
-        "idamWeb": "https://hmcts-access.service.gov.uk",
-        "rdProfessionalApi": "http://rd-professional-api-prod.service.core-compute-prod.internal",
-        "s2s": "http://rpe-service-auth-provider-prod.service.core-compute-prod.internal",
-        "feeAndPayApi": "https://payment-api-prod.service.core-compute-prod.internal",
-        "termsAndConditions": "http://xui-terms-and-conditions-aat.service.core-compute-aat.internal"
+    "referrerPolicy": {
+      "policy": "origin"
     },
-    "sessionSecret": "secretSauce",
-    "jurisdictions": [ "SSCS", "AUTOTEST1", "DIVORCE", "PROBATE", "PUBLICLAW", "bulkscan", "BULKSCAN","IA", "EMPLOYMENT", "CMC" ],
-    "feature": {
-      "appInsightsEnabled": true,
-      "proxyEnabled": false,
-      "secureCookieEnabled": true,
-      "termsAndConditionsEnabled": false,
-      "helmetEnabled": true
+    "noCache": {
+      "enable": true
     },
-    "helmet": {
-      "contentSecurityPolicy": {
-        "directives": {
-          "defaultSrc": ["'self'"],
-          "fontSrc": ["'self'", "https://fonts.gstatic.com", "data:"],
-          "styleSrc": [
-            "'self'",
-            "'unsafe-inline'",
-            "https://fonts.googleapis.com",
-            "https://fonts.gstatic.com"
-          ],
-          "scriptSrc": [
-            "'self'",
-            "'unsafe-inline'",
-            "'unsafe-eval'",
-            "www.google-analytics.com",
-            "www.googletagmanager.com",
-            "az416426.vo.msecnd.net"
-          ],
-          "connectSrc": [
-            "'self'",
-            "*.gov.uk",
-            "dc.services.visualstudio.com"
-          ],
-          "mediaSrc": ["'self'"],
-          "objectSrc": ["'self'"],
-          "frameAncestors": ["'none'"],
-          "imgSrc": [
-            "'self'",
-            "data:",
-            "https://www.google-analytics.com",
-            "https://www.googletagmanager.com",
-            "https://raw.githubusercontent.com/hmcts/"
-          ]
-        }
-      },
-      "referrerPolicy": {
-        "policy": "origin"
-      },
-      "noCache": {
-        "enable": true
-      },
-      "frameguard": {
-        "action":"deny"
-      },
-      "hidePoweredBy": {
-        "setTo": "XUI Server 1.0"
-      }
+    "frameguard": {
+      "action": "deny"
+    },
+    "hidePoweredBy": {
+      "setTo": "XUI Server 1.0"
     }
+  },
+  "redis": {
+    "host": "xui-ao-webapp-prod.redis.cache.windows.net",
```

Comment:
@paddybasi  should this be xui-mo ..instead of xui-ao...

Reply:
done